### ISSUE TITLE
May 17 Carto Rebase

### ISSUE THREAD

**Issue by jjmata** [splashblot/dronedb#17]
Every month we should:

  - [x] Fast-forward the master branch to pick up latest changes
  - [x] Read [NEWS.md](https://github.com/CartoDB/cartodb/blob/master/NEWS.md) in all relevant repos ([here](https://github.com/CartoDB/cartodb.js/blob/develop/NEWS.md) and [here](https://github.com/CartoDB/crankshaft/blob/develop/NEWS.md) and [here](https://github.com/CartoDB/Windshaft-cartodb/blob/master/NEWS.md) and [here](https://github.com/CartoDB/CartoDB-SQL-API/blob/master/NEWS.md) as well ... probably more) and adjust as necessary
  - [ ] Pick up any necessary changes to config files ([app_config.yml](https://github.com/cartodb/cartodb/blob/master/config/app_config.yml.sample) and [here](https://github.com/CartoDB/CartoDB-SQL-API/tree/master/config/environments) and [here](https://github.com/CartoDB/Windshaft-cartodb/tree/master/config/environments) as well)
  - [x] Check `/diagnosis` and fix any regressions
  - [x] Peek at sverhoeven's repo for any lessons learned
  - [x] Rebase the [tileo branch](https://github.com/splashblot/dronedb/tree/tileo)
  - [x] Rebase any other bits that we are working on but have not made the [tileo branch](https://github.com/splashblot/dronedb/tree/tileo) yet ([tileo-devops](https://github.com/splashblot/dronedb/tree/tileo-devops) tends to have stuff "in progress" that hasn't been merged to [tileo](https://github.com/splashblot/dronedb/tree/tileo) yet)
  - [x] Deploy the sucker to `alpha.tileo.co` and test some of the sample data visualizations! ✌️ 
  - [x] Rebase `tileo-develop` for new/WIP feature work
- [x] Promote "the build" to `beta.tileo.co`
  
May-specific tasks:

  - [ ] Make sure infowindows work in `*.tileo.co`
  - [ ] Pick up dataservices configuration example, see [here](https://github.com/CartoDB/cartodb/commit/bdb202cfc53d50316698331bc37096617af8fe56)
  - [ ] Re-wire DO config up so datasets show up again
  - [x] Make sure `crankshaft` is built again (was taken out [here](https://github.com/splashblot/docker-dronedb/commit/96da65257cd2ead04886cd61ddac78f4247d960f))
  - [ ] Fix/upload NPM package to use for uploading data to `tileo.co` instead of `carto.com`, see [here](https://github.com/splashblot/cartodb-nodejs/commit/af6292c5dd2e23258e57c2b9e5bca54222a9f6c1)
  - [ ] https://github.com/sverhoeven/docker-cartodb/commit/bbf2c543256ca4ef0f84099b9a680ded6108e756

**Comment by jjmata** [comment]
Start this towards the end of a milestone, so the new one can begin with a fresh rebase.

**Comment by jjmata** [comment]
Starting on this next week, other items to consider this month:

  - [ ] Try to deploy to a more powerful machine to improve performance
  - [ ] Try to enable / force `SPDY`

Thoughts, @apercas?

**Comment by jjmata** [comment]
Started on this yesterday, already have a new `docker-dronedb` that uses latest Node.js apps (and v6 node, new `mapnik`!)

**Comment by jjmata** [comment]
Turns out `crankshaft` was just in need of an update to a more current version.  Working again!  At least some of the workflows that is ... 🙄 

**EDIT:** forgot from back when I made this work that the extension needs to be activated on every account:

```
CREATE EXTENSION crankshaft WITH VERSION 'dev';
GRANT USAGE ON SCHEMA cdb_crankshaft TO publicuser;
```

[Asked a question](https://groups.google.com/forum/#!topic/cartodb/iJrZP4Hl0sk) on the Google Group to see if there is an "easy fix" for this annoyance.
